soepy/test/random_init.py

```diff
@@ -94,10 +94,25 @@ def random_init(constr=None):
         "PARAMETERS"
     ]["delta_s3"] = np.random.uniform(0.1, 0.9, 3).tolist()
 
-    init_dict["PARAMETERS"]["theta_p"], init_dict["PARAMETERS"][
-        "theta_f"
+    init_dict["PARAMETERS"]["const_p"], init_dict["PARAMETERS"][
+        "const_f"
     ] = np.random.uniform(0.5, 5, 2).tolist()
 
+    # Random number of types: 1, 2, 3, 4, or 5
+    num_types = 1
+    # num_types = int(np.random.choice([1, 2, 3, 4, 5], 1))
+
+    for i in range(1, num_types):
+        # Draw random parameters
+        init_dict["PARAMETERS"]["theta_p" + str(i)], init_dict["PARAMETERS"][
+            "theta_f" + str(i)
```

Comment:
Please do not use the `str` function at all and instead use proper `.format()` syntax everywhere in the code. https://pyformat.info/

Reply:
Done